chain/contracts/namespaces.go

```diff
@@ -30,7 +30,7 @@ var (
123565b50505050505092915050565b600082601f83011261116f578081fd5b813567ffffffffffffffff811115611185578182fd5b611198601f8201601f191660200161165a565b91508082528360208285010111156111af57600080fd5b8060208401602084013760009082016020015292915050565b803561ffff8116811461083957600080fd5b6000602082840312156111eb578081fd5b81356111f6816116e6565b9392505050565b6000806040838503121561120f578081fd5b823561121a816116e6565b9150602083013561122a816116d1565b809150509250929050565b60008060408385031215611247578182fd5b8235611252816116e6565b9150602083013567ffffffffffffffff81111561126d578182fd5b6112798582860161115f565b9150509250929050565b600080600080600060a0868803121561129a578081fd5b6112a487876111c8565b945060208087013567ffffffffffffffff808211156112c1578384fd5b6112cd8a838b0161115f565b965060408901359150808211156112e2578384fd5b6112ee8a838b0161115f565b95506060890135915080821115611303578384fd5b818901915089601f830112611316578384fd5b81356113246110f982611681565b81815284810190848601875b84811015611359576113478f8984358a010161115f565b84529287019290870190600101611330565b509097505050506080890135925080831115611373578384fd5b5050611381888289016110db565b9150509295509295909350565b6000806000606084860312156113a2578283fd5b83356113ad816116e6565b92506020840135915060408401356113c4816116d1565b809150509250925092565b6000806000606084860312156113e3578283fd5b6113ed85856111c8565b925060208401359150604084013567ffffffffffffffff81111561140f578182fd5b61141b8682870161115f565b9150509250925092565b6000815180845261143d8160208601602086016116a1565b601f01601f19169290920160200192915050565b600082516114638184602087016116a1565b9190910192915050565b6001600160a01b0392909216825261ffff16602082015260400190565b901515815260200190565b6000608082526114a86080830187611425565b6020838203818501526114bb8288611425565b848103604086015286518082529092508183019082810284018301838901865b8381101561150957601f198784030185526114f7838351611425565b948601949250908501906001016114db565b5050868103606088015287518082529084019450915050818601845b8281101561154a5781516001600160a01b031685529383019390830190600101611525565b50929998505050505050505050565b60006040825261156c6040830185611425565b905061ffff831660208301529392505050565b60208082526011908201527037b7363ca1b7b73a3930b1ba27bbb732b960791b604082015260600190565b6020808252600d908201526c092dcecc2d8d2c840d2dcc8caf609b1b604082015260600190565b602080825260129082015271092dcc8caf05ad6caf240dad2e6dac2e8c6d60731b604082015260600190565b6020808252600b908201526a26bab9ba103234b33332b960a91b604082015260600190565b6020808252600f908201526e105b1c9958591e481c1c995cd95b9d608a1b604082015260600190565b61ffff91909116815260200190565b60405181810167ffffffffffffffff8111828210171561167957600080fd5b604052919050565b600067ffffffffffffffff821115611697578081fd5b5060209081020190565b60005b838110156116bc5781810151838201526020016116a4565b838111156116cb576000848401525b50505050565b6001600160a01b038116811461108757600080fd5b61ffff8116811461108757600080fdfea264697066735822122083dd59a17980180260f9a47cb0d1d0b5fd00415c3dafa67084542b2593a7a5e964736f6c634300060c0033"
```

Comment:
To me it makes more sense to add it in base64 instead than hex. Is that possible?

Reply:
this is generated code, the only thing that changes is the pkg name to be unified between all contracts.